src/components/DownloadTrajectoryMenu/index.tsx

```diff
@@ -53,22 +53,33 @@ const DownloadTrajectoryMenu = ({
         downloadFile(simulariumFile.name);
     };
 
+    const isDisabled = () => {
```

Comment:
nit: couldn't this just be a regular boolean const, instead of a function? like:

`const isDisabled = !fileIsLoaded() || isBuffering;`

Just feels a little more concise, since it's not taking in any arguments and is just returning a bool